### PR TITLE
BUG: fix composer comparison hack.

### PR DESCRIPTION
**Pull request by cournape**
@johntyree 

I added basic instructions in the `README.rst`. Let me know if that does not work for you.

### CODE REVIEW

**Comment by johntyree** [comment]
Needs `jinja2` added to dev_requirements.

File: README.rst [hunk 1]
```diff
@@ -32,7 +46,28 @@ Known not to work:
...
+=============================
+
+First, clone composer's somewhere on your machine::
+
+    git clone https://github.com/composer/composer
+
+Then, use the `scripts/scenario_to_php.py` script to write a php file that will
+print the composer's solution for a given scenario::
+
+    python --composer-root <path to composer github checkout> \
```
Comment (johntyree) [inline README.rst]: The `--composer-root` option apparently needs to come after the `scenario_to_php` script, and that script is a `.py` script, not a `.php` script.

**Comment by johntyree** [comment]
This works nicely.

We can discuss whether or not composer does what we want in another issue.

**Comment by johntyree** [comment]
Fix the missing requirement and README code and this LGTM

**Comment by johntyree** [comment]
@cournape

**Comment by cournape** [comment]
Anything to add ? Otherwise I will merge it

**Comment by johntyree** [comment]
No this is ready from my point of view. Works nicely.

**Pull request merged**